src/test/java/DataManipulationTest.java

```diff
@@ -0,0 +1,16 @@
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Avoid * imports

Reply:
This import was added by the IDE. Removed in the last commit.